solc/CommandLineInterface.cpp

```diff
@@ -1097,6 +1097,21 @@ void CommandLineInterface::outputCompilationResults()
 {
 	solAssert(m_options.input.mode == InputMode::Compiler || m_options.input.mode == InputMode::CompilerWithASTImport, "");
 
+	std::set<std::string> uncompiledContracts;
+	for (auto const& contract: m_compiler->contractNames())
+		if (objectWithLinkRefsHex(m_compiler->object(contract)).empty())
```

Comment:
`objectWithLinkRefsHex()` does extra processing you don't need (hex-encoding, listing link references).
```suggestion
		if (m_compiler->object(contract).bytecode.empty())
```